admin-dev/backup.php

```diff
@@ -108,7 +108,7 @@
 
 fclose($fp);
 
-if ($ret === false) {
+if ((int) $ret === 0) {
```

Comment:
```suggestion
if ($ret === 0) {
```

`fpassthru` return an `int`

Reply:
`fpassthru` can also return false

Reply:
I see nothing related to boolean return in the documentation: https://www.php.net/fpassthru

Reply:
I think it changed in PHP 8, but it was possible before. http://web.archive.org/web/20180223045536/https://www.php.net/manual/en/function.fpassthru.php:
>If an error occurs, fpassthru() returns FALSE. Otherwise, fpassthru() returns the number of characters read from handle and passed through to the output.

Reply:
OMFG, sad story, we have to be very careful now when using the PHP documentation :( 
I was wondering what the new return when the function's call failed :thinking: